prow/github/app_auth_roundtripper.go

```diff
@@ -117,7 +117,8 @@ var TimeNow = func() time.Time {
 
 func (arr *appsRoundTripper) addAppAuth(r *http.Request) *appsAuthError {
 	now := TimeNow()
-	expiresAt := now.Add(10 * time.Minute)
+	// GitHub's clock may lag a few seconds, so we do not use 10min here.
+	expiresAt := now.Add(9 * time.Minute)
```

Comment:
I'm not very familiar with this code. How does this token get replaced? This will cause us to consider the token invalid when it is still valid so I want to be sure that it will be properly replaced with a fresh token at that time or earlier.

Reply:
Ah, gotcha. Thanks for the explanation.
/approve